modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataLayoutResourceTest.java

```diff
@@ -398,7 +447,7 @@ protected DataLayout testGetDataLayout_addDataLayout() throws Exception {
 
 	@Override
 	protected DataLayout
-			testGetSiteDataLayoutByContentTypeByDataLayoutKey_addDataLayout()
+	testGetSiteDataLayoutByContentTypeByDataLayoutKey_addDataLayout()
```

Comment:
Can you run gw formatSource on data-engine-rest-test? I think it will fix it.

Reply:
This is weird. I didn't change this :thinking: . Check out below how it is formatted in my local branch:

![image](https://user-images.githubusercontent.com/6429921/107217159-8a64eb00-69ec-11eb-9df3-5e2b3e23be5f.png)